cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -484,7 +485,7 @@ export class ArmatureDisplay extends UIRenderer {
      * @en Draw call list.
      * @zh Draw call 列表。
      */
-    get drawList (): RecyclePool<ArmatureDisplayDrawData> { return this._drawList; }
+    get drawList (): memop.RecyclePool<ArmatureDisplayDrawData> { return this._drawList; }
```

Comment:
Will it break compatibility? Now the function is in `memop` module.

Reply:
no，we still keep them exporting, we handle the compatibility here:
https://github.com/cocos/cocos-engine/blob/d2926635587f516f0f3f54275abc81d3dab90fd8/cocos/base/utils/src/index.ts#L7-L8

the deprecate info is here 
https://github.com/cocos/cocos-engine/blob/bb8fd4b0307a3e0cf25b16cfea4253c08a1d8a47/cocos/core/deprecated-3.9.0.ts

we've incorrctly  exported the interface `Pool`, `RecyclePool` and  `CachedArray` directly, they should be deprecated and developers can access these 3 interfaces in the `mamop` namespace.

Reply:
for project
```
// deprecated
import { Pool, RecyclePool and CachedArray } from 'cc';

// recomanded

import { memop } from 'cc';
memop.Pool
//...
```